Directory.Build.props

```diff
@@ -1,29 +1,15 @@
-<Project>
-  <Import
-    Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), AspNetCoreSettings.props))\AspNetCoreSettings.props"
-    Condition=" '$(CI)' != 'true' AND '$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), AspNetCoreSettings.props))' != '' " />
-
-  <Import Project="version.props" />
-  <Import Project="build\dependencies.props" />
-
-  <PropertyGroup>
-    <Product>Microsoft ASP.NET Core Blazor</Product>
-    <RepositoryUrl>https://github.com/aspnet/blazor</RepositoryUrl>
-    <RepositoryType>git</RepositoryType>
-    <RepositoryRoot>$(MSBuildThisFileDirectory)</RepositoryRoot>
-    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)build\Key.snk</AssemblyOriginatorKeyFile>
-    <SignAssembly>true</SignAssembly>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <AssemblySigningCertName>Microsoft</AssemblySigningCertName>
-    <LangVersion>7.3</LangVersion>
-
-    <!-- Disable API baseline checks. -->
-    <EnableApiCheck>false</EnableApiCheck>
-  </PropertyGroup>
-
-  <Import Project="build\arcade.props" />
-
-  <ItemGroup>
-    <PackageReference Include="Internal.AspNetCore.Sdk" PrivateAssets="All" Version="$(InternalAspNetCoreSdkPackageVersion)" />
-  </ItemGroup>
-</Project>
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+
+  <PropertyGroup>
+    <Product>Microsoft ASP.NET Core Blazor</Product>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <PackageProjectUrl>https://asp.net/</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/aspnet/Blazor</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <StrongNameKeyId>MicrosoftAspNetCore</StrongNameKeyId>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <LangVersion>7.3</LangVersion>

```

Comment:
Why are we specifying 7.3? Why not 8.0?

Reply:
This is the value that was here before this change. Git is just showing a diff because I deleted unnecessary code that was on the lines before and after. @SteveSandersonMS if you want a higher lang version, you're welcome to change it.